tests/integration/timeseries/test_par.py

```diff
@@ -47,3 +49,16 @@ def test_par():
     assert sampled.shape == data.shape
     assert (sampled.dtypes == data.dtypes).all()
     assert (sampled.notnull().sum(axis=1) != 0).all()
+
+
+def test_column_after_date():
+    """Test that adding columns after the `sequence_index` column works."""
+    date = datetime.datetime.strptime('2020-01-01', '%Y-%m-%d')
+    daily_timeseries = pd.DataFrame({
```

Comment:
I think it would be worth to make this test slightly more complex, so there are multiple data types and both entity columns and context columns.